pages/standard/bulkdnaseq.tsx

```diff
@@ -1,51 +1,24 @@
 import React from 'react';
-import HtanNavbar from '../../components/HtanNavbar';
-import Footer from '../../components/Footer';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import Breadcrumb from 'react-bootstrap/Breadcrumb';
-import { GetServerSideProps, GetStaticProps } from 'next';
-import fetch from 'node-fetch';
-import { CmsData } from '../../types';
-import { WORDPRESS_BASE_URL } from '../../ApiUtil';
+import { GetStaticProps } from 'next';
 
-export interface BulkDNASeqProps {
-    data: CmsData[];
-}
+import DataStandard, { DataStandardProps } from "../../components/DataStandard";
+import { getStaticContent } from '../../ApiUtil';
+import { getDataSchema } from "../../lib/dataSchemaHelpers";
 
-function BulkDNASeq(data: BulkDNASeqProps) {
+const BulkDNASeq: React.FunctionComponent<DataStandardProps> = props => {
     return (
-        <>
-            <HtanNavbar />
-            <Container>
-                <Row>
-                    <Breadcrumb className="mt-3">
-                        <Breadcrumb.Item href="/">Home</Breadcrumb.Item>
-                        <Breadcrumb.Item href="/standards">
-                            Data Standards
-                        </Breadcrumb.Item>
-                        <Breadcrumb.Item active>Bulk DNA Seq</Breadcrumb.Item>
-                    </Breadcrumb>
-                </Row>
-                <Row>
-                    <span
-                        dangerouslySetInnerHTML={{
-                            __html: data.data[0].content.rendered,
-                        }}
-                    ></span>
-                </Row>
-            </Container>
-            <Footer />
-        </>
+        <DataStandard {...props} title="Bulk DNA Seq" />
     );
 }
 
 export const getStaticProps: GetStaticProps = async (context) => {
-    let slugs = ['data-standards-bulk-dnaseq-blurb'];
-    let overviewURL = `${WORDPRESS_BASE_URL}${JSON.stringify(slugs)}`;
-    let res = await fetch(overviewURL);
-    let data = await res.json();
-    return { props: { data } };
+    const data = await getStaticContent(['data-standards-bulk-dnaseq-blurb']);
+    // TODO current schema file does not have any id matching dnaseq
```

Comment:
@inodb Couldn't seem to find any id for dnaseq. Are we missing it or does it have a different name?

Reply:
@onursumer it's prolly still called WholeExomeSeq or WES

Reply:
Thanks, WES it is!